src/vcs_scanner/secret_scanners/cli.py

```diff
@@ -60,10 +60,16 @@ def create_cli_argparser() -> ArgumentParser:
                                envvar="RESC_EXIT_CODE_BLOCK",
                                help="Exit code given if CLI encounters findings tagged with Block, default 1. "
                                     "Can also be set via the RESC_EXIT_CODE_BLOCK environment variable")
-    parser_common.add_argument("--filter-tag", required=False, action=EnvDefault, type=str,
```

Comment:
This is a breaking change, per definition this requires a major new release. Why not keep the `--filter-tag` or make it backward compatible as in, if `--filter-tag` is provided then fill `--include-tags` with it?